src/app/providers/ContextProviders.tsx

```diff
@@ -25,15 +26,17 @@ export const ContextProviders = ({ children }: Props) => {
         <QueryClientProvider client={queryClient}>
           <AlertProvider>
             <MigrationProvider>
-              <BuilderContextProviderWithPrices>
-                <BoosterProvider>
-                  <AllocationsContextProvider>
-                    <BalancesProvider>
-                      <MainContainer>{children}</MainContainer>
-                    </BalancesProvider>
-                  </AllocationsContextProvider>
-                </BoosterProvider>
-              </BuilderContextProviderWithPrices>
+              <HeroCollapseProvider>
```

Comment:
I'm not sure if it makes sense to use it directly in the MainContainer to restrict it where used or here for visibility

Reply:
For now it makes sense to keep it there for visibility.